pkg/cloud/service.go

```diff
@@ -95,16 +95,13 @@ func (s *Service) pollOperation(ctx context.Context, op operation) error {
 		pollCount++
 		klog.V(5).Infof("op.isDone(%v) waiting; op = %v, poll count = %d", ctx, op, pollCount)
 		s.RateLimiter.Accept(ctx, op.rateLimitKey())
-		done, err := op.isDone(ctx)
-		if err != nil {
-			klog.V(5).Infof("op.isDone(%v) error; op = %v, poll count = %d, err = %v, retrying", ctx, op, pollCount, err)
-		}
-
-		if done {
-			break
+		switch done, err := op.isDone(ctx); {
```

Comment:
switch seems extraneous:
```
done, err := op.isDone(ctx)
if err != nil {
  ...
  return ...
}
if done {
  ...
  return ...
}

```

Reply:
Seems like a pretty canonical use of switch over a sequence of ifs, eg. https://golang.org/doc/effective_go.html#switch

Reply:
sgtm